app/.gitlab-ci.yml

```diff
@@ -279,6 +279,21 @@ test:frontend:
       changes:
       - app/**/*
 
+test:frontend-prettier:
+  needs: ["build:frontend"]
+  stage: test
+  image: $FRONTEND_TAG
+  inherit:
+    # no docker login
+    default: false
+  script:
+    - cd /app
+    - npx prettier --check src/*
```

Comment:
I think it's better to use the same version of Prettier already installed in the dev dependencies, since Prettier are pretty aggressive at changing the formatting rules between even non-major versions, so using `npx` will likely introduce discrepancies (since it tends to use the latest version) from what we got installed on the image.

Reply:
(ie `yarn prettier --args`)

Reply:
> (ie `yarn prettier --args`)

aaah, I'm used to npm where `npm prettier --args` wouldn't work. But it seems yarn is smarter and knows we actually want to use a locally installed module.

I'll change it, thanks!